robot_skills/src/robot_skills/force_sensor.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import rospy
+from geometry_msgs.msg import WrenchStamped
+
+
+class TimeOutException(Exception):
+    pass
+
+
+class ForceSensor(object):
+    def __init__(self, topic, force_norm_threshold=2.5):
+        """
+        Class for conveniently using a force sensor
+
+        :param topic: Topic to use for WrenchStamped measurement
+        :param force_norm_threshold: Edge up if norm exceeds this value (defaults to 2.5)
```

Comment:
2.5 what? Newton?

Reply:
You sound just like my Physics teacher. "What are those? Newtons? Joules? Bananas?"

Reply:
It uses a `Wrench` message which should adhere to https://www.ros.org/reps/rep-0103.html#units